Cargo.toml

```diff
@@ -1,5 +1,4 @@
 [workspace]
 members = [
-  "libursa",
-  "libzmix"
+  "libursa"
```

Comment:
Did you mean to remove libzmix? I've not been active on the conversation for the last month or so. I'm just curious how this is getting reorganized.  I'm looking at your branch and I still see a libzmix folder.

Reply:
I just removed it from the workspaces for now because many people clone the repo and run `cargo build` and wonder why it doesn't compile. Until we fix libzmix I'd rather not deal  with the questions and fuss.